### PR TITLE
Better .gitignore file

### PR DESCRIPTION
**Pull request by fourlastor**
the other broke everything

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,8 +1,52 @@
...
+**/.idea/libraries
+**/.idea/dictionaries
+**/.idea/runConfigurations.xml
+
+# OSX and Windows
+*.DS_Store
+Thumbs.db
+
+# Generated during build
+com_crashlytics_export_strings.xml
```
Comment (rock3r) [inline .gitignore]: Do we even use Crashlytics?

Comment (fourlastor) [inline .gitignore]: I copied yours

**Comment by rock3r** [comment]
Minor comment then LGTM 👍

File: .gitignore (hunk 1 above)
Comment (rock3r) [inline .gitignore]: NEVER TRUST ME!!!1!

Comment (fourlastor) [inline .gitignore]: 7744c2c

**Comment by rock3r** [comment]
#shipit

**Pull request merged**